build.gradle

```diff
@@ -55,19 +55,13 @@ allprojects {
 	dependencies {
 		minecraft "com.mojang:minecraft:$Globals.mcVersion"
 		mappings "net.fabricmc:yarn:${Globals.mcVersion}${Globals.yarnVersion}:v2"
-		modCompile "net.fabricmc:fabric-loader:0.7.2+build.174"
```

Comment:
Either revert this change or update every single fabric.mod.json's dependencies.